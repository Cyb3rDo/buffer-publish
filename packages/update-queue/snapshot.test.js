// use story.js files as snapshots
import initStoryshots from 'storyshots';

initStoryshots({
  suit: 'Snapshots',
});
