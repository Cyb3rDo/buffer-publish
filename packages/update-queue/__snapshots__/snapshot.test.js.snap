// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots UpdateQueue UpdateQueue 1`] = `
<ul
  className="ul"
>
  <li
    className="li"
  >
    <div
      className="card"
      onMouseEnter={undefined}
      onMouseLeave={undefined}
    >
      {"id":"4ec93ae4512f7e6307000003","created_at":1320703583,"day":"Monday 7th November","due_at":1320543483,"due_time":"07:02 pm","profile_id":"4eb854340acb04e870000010","profile_service":"twitter","status":"buffer","text":"This is me in an alternate life where i can breakdance again j.mp/w...","text_formatted":"This is me in an alternate life where i can breakda...","user_id":"4eb9276e0acb04bb81000067","via":"firefox"}
    </div>
  </li>
  <li
    className="li"
  >
    <div
      className="card"
      onMouseEnter={undefined}
      onMouseLeave={undefined}
    >
      {"id":"4ec93ae4512f7e6307000002","created_at":1320703582,"day":"Monday 7th November","due_at":1320543480,"due_time":"07:01 pm","profile_id":"4eb854340acb04e870000010","profile_service":"twitter","status":"buffer","text":"This is me in an alternate life where i can breakdance j.mp/w...","text_formatted":"This is me in an alternate life where i can breakda...","user_id":"4eb9276e0acb04bb81000067","via":"firefox"}
    </div>
  </li>
</ul>
`;
